hazelcast/src/test/java/com/hazelcast/map/SizeEstimatorTest.java

```diff
@@ -57,7 +57,7 @@ public void testSinglePut() throws InterruptedException {
 
     @Test
     public void testExactHeapCostAfterUpdateWithMultipleBackupNodes() throws InterruptedException {
-        long expectedPerEntryHeapCost = 172L;
+        long expectedPerEntryHeapCost = 156L;
```

Comment:
Yes it depends on JVM settings, there is a merged PR to fix that issue for one of 3.5 branches, i will also port it to master.
https://github.com/hazelcast/hazelcast/commit/b90f8dbd9ebac29a02ecfe6c6f44bc1a1e62269d